Backend/config/unittest.cfg

```diff
@@ -1,2 +1,13 @@
 [unittest]
-start-dir = Backend
\ No newline at end of file
+start-dir = .
+[coverage]
+always-on = True
+coverage-report = html
+coverage =
+  mentii
+  utils
+  problems
+omit =
+  mentii/__init__.py
+  utils/__init__.py
+  problems/__init__.py
```

Comment:
@smc395 yeah i saw that. The omit option was just for coverage.py directly. I couldn't get it working with nose2. Ill update the config file to remove these couple lines